This patch is based on the upstream commit described below, adapted for use
in the Debian package by Peter Michael Green.

commit d5a01dd3a56495cd1f19883f875cede910ce786e
Author: Juniper Tyree <50025784+juntyr@users.noreply.github.com>
Date:   Sun Aug 6 15:34:23 2023 +0300

    Update adjacent enum non-roundtrip tests to match serde-rs/serde#2505 (#471)
    
    Update adjacent enum non-roundtrip tests to match https://github.com/serde-rs/serde/pull/2505

--- a/tests/123_enum_representation.rs
+++ b/tests/123_enum_representation.rs
@@ -109,3 +109,3 @@
     };
-    let e = "(type:\"VariantA\",content:(foo:1,bar:2,different:Foo))";
+    let e = "(type:VariantA,content:(foo:1,bar:2,different:Foo))";
     test_ser(&v, e);
@@ -116,3 +116,3 @@
     let v = EnumStructAdjacently::VariantB { foo: 1, bar: 2 };
-    let e = "(type:\"VariantB\",content:(foo:1,bar:2))";
+    let e = "(type:VariantB,content:(foo:1,bar:2))";
     test_ser(&v, e);
@@ -176,3 +176,3 @@
 fn test_adjacently_a_de() {
-    let s = "(type:\"VariantA\",content:(foo:1,bar:2,different:Foo))";
+    let s = "(type:VariantA,content:(foo:1,bar:2,different:Foo))";
     let e = EnumStructAdjacently::VariantA {
@@ -187,3 +187,3 @@
 fn test_adjacently_b_de() {
-    let s = "(type:\"VariantB\",content:(foo:1,bar:2))";
+    let s = "(type:VariantB,content:(foo:1,bar:2))";
     let e = EnumStructAdjacently::VariantB { foo: 1, bar: 2 };
--- a/tests/207_adjacently_tagged_enum.rs
+++ b/tests/207_adjacently_tagged_enum.rs
@@ -16,3 +16,3 @@
 
-    assert_eq!(ron_string, "(type:\"Index\",data:1)");
+    assert_eq!(ron_string, "(type:Index,data:1)");
 
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,3 +53,3 @@
 [dependencies.serde]
-version = "1.0.60"
+version = "1.0.181"
 features = ["serde_derive"]
